src/_scss/pages/about/_contentWrap.scss

```diff
@@ -0,0 +1,18 @@
+.about-content {
+    background-color: $color-white;
+    box-shadow: $container-shadow;
+    color: $color-base;
+    border-top: 1px solid $color-gray-border;
+    border-right: 1px solid $color-gray-border;
+    border-bottom: 1px solid $color-gray-border;
+
+    .about-padded-content {
+        padding: rem(35);
+    }
+
+    @media(min-width: $large-screen) {
```

Comment:
Please use the standard media mixin we're using everywhere else on the site `@include media($screen-var)`

Reply:
Done